students/BrianNisonger/Lesson5/assignment/database.py

```diff
@@ -0,0 +1,108 @@
+import os
+import csv
+import logging
+from pymongo import MongoClient
+
+
+class MongoDBConnection(object):
+    def __init__(self, host='127.0.0.1', port=27017):
+        self.host = host
+        self.port = port
+        self.connection = None
+
+    def __enter__(self):
+        self.connection = MongoClient(self.host, self.port)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.connection.close()
+
+
+def calculate_availability(product_list, rental_list):
+    for product in product_list:
+        count = 0
+        for rental in rental_list:
+            if rental["product_id"] == product["product_id"]:
+                count += 1
```

Comment:
A doc string would be useful here, and everywhere. :) You can use sum with a comprehension:
```
count = sum(1 for rental in rental_list if rental["product_id"] == product["product_id"])
```